book/03-hacking-atom/1-hacking-atom.asc

```diff
@@ -27,6 +27,8 @@ include::sections/A03-textmate.asc[]
 
 === Summary
 
-( List of topics we covered / appendix? )
+If you finished this chapter, you should be an Atom-hacking master. We've discussed how you should work with CoffeeScript, and how to put it to good use in creating packages. You should also be able to do this in your own created theme now.
```

Comment:
In summaries such as this, I tend to avoid "you should" statements. If the manual didn't explain things well enough, it may be the fault of the manual, not the reader. "You should" statements imply that the fault lies with the reader.

I'd recommend sticking to "we discussed", "we outlined", "we described" statements. Example: "We discussed how to work with CoffeeScript and how it can be used in creating packages and themes."